framework/captcha/GdDriver.php

```diff
@@ -24,7 +24,8 @@ public function init()
     {
         parent::init();
 
-        if (!extension_loaded('gd') || ($gdInfo = gd_info() && empty($gdInfo['FreeType Support']))) {
+        // Additional brackets for fix possible bug with operator precedence
```

Comment:
Comment isn't necessary. Change itself is OK.

Reply:
Comment removed